### PR TITLE
Ports a Boatload of Teshari Customization from CitRP, Chomp & YW, Small Update to Protogen Markings

### PR DESCRIPTION
**Pull request by KasparoVy**
All changes to the standard non _vr eyes, gloves, suit, undercoat/deptuniform and ties.dmi are due to be ported upstream to Polaris
The orange goggles are going to be ported to Polaris

This PR will be referred to from Polaris repo re: the assets going their way then this will be marked ready for review

Changes:
Adds Teshari-fitted collar sprites (with refitting logic from YawnWider).
Adds Teshari-fitted sleek orange goggles (from CitRP).
Adds Teshari-fitted rings (from CitRP).
Adds better Teshari-fitted Space Suit & Helmet sprites (by KBraid).
Adds better Teshari-fitted Radiation Suit & Helmet sprites (by KBraid).
Adds full sets of Teshari standard & job cloaks from CitRP (different sprites than Polaris) - job cloaks can be selected as an option when selected in loadout. Captain, Psych, RD, HoS & HoP cloaks are net-new (from CitRP).
Adds the dark retrowave & black + glowing Teshari cloaks (from Chomp).
Adds full sets of Teshari standard & job belted cloaks (from Chomp).
Adds full set of Teshari belted cloaks (from Chomp).
Adds better Teshari standard & job undercoat sprites (from CitRP).
Further enhances colouration of protogen markings.

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_xeno.dm [hunk 1]
```diff
@@ -270,86 +270,222 @@
 	path = /obj/item/clothing/suit/storage/seromi/cloak/jobs/ce
 	allowed_roles = list("Chief Engineer")
 
+/datum/gear/suit/dept/cloak/ce/New() //VS EDIT
```
Comment (Novacat) [inline code/modules/client/preference_setup/loadout/loadout_xeno.dm:273]: I believe there is already a loadout_xeno_vr you can put these in.

File: code/modules/clothing/suits/aliens/seromi_cit.dm [hunk 2]
```diff
@@ -0,0 +1,275 @@
+// Standard Cloaks
+/obj/item/clothing/suit/storage/teshari/cloak_cit
```
Comment (Novacat) [inline code/modules/clothing/suits/aliens/seromi_cit.dm:2]: all the _cit references should probably be renamed to _vr, in the filenames too.

File: code/modules/clothing/under/accessories/accessory_vr.dm [hunk 3]
```diff
@@ -31,6 +64,39 @@
 	icon_override = 'icons/mob/ties_vr.dmi'
 	icon_state = "collar_blk"
 	var/writtenon = 0
+	var/icon_previous_override //yw addition
```
Comment (Novacat) [inline code/modules/clothing/under/accessories/accessory_vr.dm:67]: likewise, Vorestation Edit should be used here.

**Review by Novacat — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/accessories/accessory_vr.dm (hunk 3 above)
Comment (KasparoVy) [reply code/modules/clothing/under/accessories/accessory_vr.dm:67]: Was just noting where I got this from as it's already a _VR file

File: code/modules/clothing/suits/aliens/seromi_cit.dm (hunk 2 above)
Comment (KasparoVy) [reply code/modules/clothing/suits/aliens/seromi_cit.dm:2]: Was just noting where they came from but no problem, will simply leave a comment

File: code/modules/client/preference_setup/loadout/loadout_xeno.dm (hunk 1 above)
Comment (KasparoVy) [reply code/modules/client/preference_setup/loadout/loadout_xeno.dm:273]: I opted to put them here so folks didn't have to uump between files to see the code that gives these loadout items a choice bw Reggie and CitRP versions, but will split them off anyway per your request

**Comment by Novacat** [comment]
The big reason I request you change the edits is because both YW and CitRP are downstreams of us, and it would mess them up if we started using their edit markers for files. Much like if Polaris started adding //VOREStation Edit to their files it would mess us up.

**Comment by KasparoVy** [comment]
Thanks for the explanation @Novacat, can see how that'd cause problems

Taking a while because I missed a few other things

File: code/modules/client/preference_setup/loadout/loadout_xeno.dm [hunk 4]
```diff
@@ -424,11 +424,16 @@
...
 	display_name = "toe-less jackboots, knee-length"
 	path = /obj/item/clothing/shoes/boots/jackboots/toeless/knee
 
 /datum/gear/shoes/toelessthigh
 	display_name = "toe-less jackboots, thigh-length"
-	path = /obj/item/clothing/shoes/boots/jackboots/toeless/thigh
\ No newline at end of file
+	path = /obj/item/clothing/shoes/boots/jackboots/toeless/thigh
+
+/datum/gear/eyes/aerogelgoggles
```
Comment (KasparoVy) [inline code/modules/client/preference_setup/loadout/loadout_xeno.dm:435]: To be sent to Polaris

File: code/modules/clothing/glasses/glasses.dm [hunk 5]
```diff
@@ -543,3 +543,38 @@ BLIND     // can't see anything
 	desc = "A set of implantable lenses designed to augment your vision"
 	icon_state = "thermalimplants"
 	item_state_slots = list(slot_r_hand_str = "sunglasses", slot_l_hand_str = "sunglasses")
+
+/obj/item/clothing/glasses/aerogelgoggles
```
Comment (KasparoVy) [inline code/modules/clothing/glasses/glasses.dm:547]: To be sent to Polaris

**Comment by KasparoVy** [comment]
Gonna do more testing to make sure it's all still kosher then do Polaris backport tmrw

IDK why Travis is mad, it compiled fine on local and the loc says it passed despite the header stating it failed https://travis-ci.com/github/VOREStation/VOREStation/jobs/441295965

**Comment by KasparoVy** [comment]
Job undercoats bust.

**Comment by KasparoVy** [comment]
OK that's all

**Comment by KasparoVy** [comment]
OK that's all, ready for further review.

Aside from the legs looking wack on all the undercoats apparently the mining, cargo + some other one's north & south dirs were skewed 1px to the left this whole time (even on Polaris).

**Review by Novacat — CHANGES_REQUESTED** [review]
I am seeing a lot of changes to Polaris icon files. Are those changes being ported upstream? Also, have you doublechecked everything to make sure it all works?

**Comment by KasparoVy** [comment]
I tested meticulously during my multiple passes and ported all changes to Polaris files in #7760

**Comment by KasparoVy** [comment]
I tested meticulously during my final passes and ported all changes to Polaris files in Polaris/7760, that's why the last couple commits exist because I caught what I missed

Made the retrowave & glowing cloaks more obviously VR

**Comment by Novacat** [comment]
I am assuming the https://github.com/PolarisSS13/Polaris/pull/7760

**Review by Novacat — APPROVED** [review]

**Comment by KasparoVy** [comment]
Yep! Cheers

**Comment by KasparoVy** [comment]
@Novacat I noticed one duplicate sprite I missed from PR https://github.com/VOREStation/VOREStation/pull/9307, rebased and sorted it out here

**Comment by KasparoVy** [comment]
Converted the last of the protogen markings to multiplicative blending, did parity PR on CitRP

**Comment by KasparoVy** [comment]
Should probably wait for https://github.com/VOREStation/VOREStation/pull/9327

**Comment by Novacat** [comment]
I plan on merging this before 9327, mostly because I plan on doing a hotfix and if this is already tested and ready I can have it live. 9327 I am skittish on because I expect it to cause unforeseen issues so I would rather leave it for a major patch rather than a hotfix.

**Comment by KasparoVy** [comment]
Fair enough, thanks!

**Comment by KasparoVy** [comment]
:O yay. Cheers

**Pull request merged**